### PR TITLE
chore: apply Biome unsafe changes

### PR DESCRIPTION
**Pull request by davidbgk**
Without `useBlockStatements`

### CODE REVIEW

File: umap/static/umap/js/modules/sync/updaters.js [hunk 1]
```diff
@@ -89,9 +89,10 @@ export class FeatureUpdater extends BaseUpdater {
       console.error(`Unable to find feature with id = ${metadata.id}.`)
     }
     switch (key) {
-      case 'geometry':
+      case 'geometry': {
```
Comment (yohanboniface) [inline umap/static/umap/js/modules/sync/updaters.js:92]: Don't get this one

File: umap/static/umap/js/umap.js [hunk 2]
```diff
@@ -1340,7 +1338,6 @@ U.Map = L.Map.extend({
         'options.overlay.url_template',
         {
           handler: 'BlurInput',
-          helpText: `${L._('Supported scheme')}: http://{s}.domain.com/{z}/{x}/{y}.png`,
           placeholder: 'url',
           helpText: L._('Background overlay url'),
```
Comment (yohanboniface) [inline umap/static/umap/js/umap.js:1342]: I think this one should be `label` and the other one kept.

File: umap/static/umap/js/modules/rules.js [hunk 3]
```diff
@@ -71,9 +71,10 @@ class Rule {
         break
       }
     }
-    if (vars.length != 2) return
+    if (vars.length !== 2) return
     this.key = vars[0]
     this.expected = vars[1]
+    // biome-ignore lint/suspicious/noGlobalIsNan: expected might not be a number.
```
Comment (yohanboniface) [inline umap/static/umap/js/modules/rules.js:77]: don't get this exception

Comment (yohanboniface) [reply umap/static/umap/js/modules/rules.js:77]: Ok, sorry, found the comment above, `Number.isNaN` excepts a number, that may be `NaN`, while `isNaN` returns false for every non number value.

**Review by yohanboniface — APPROVED** [review]

File: umap/static/umap/js/modules/sync/updaters.js (hunk 1 above)
Comment (davidbgk) [reply umap/static/umap/js/modules/sync/updaters.js:92]: The error is that one: https://biomejs.dev/linter/rules/no-switch-declarations/ maybe we should use a `if` for that one-case switch?

Comment (davidbgk) [reply umap/static/umap/js/modules/sync/updaters.js:92]: Something like that:

```suggestion
    if (key === 'geometry') {
      const datalayer = this.getDataLayerFromID(metadata.layerId)
      this.getDataLayerFromID(metadata.layerId).geoJSONToLeaflet({
        geometry: value,
        id: metadata.id,
        feature,
      })
    } else {
      this.updateObjectValue(feature, key, value)
      feature.datalayer.indexProperties(feature)
    }

```

**Pull request merged**